domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/LayoutServiceImpl.java

```diff
@@ -59,4 +52,17 @@ public void createDisplayGroups(List<DisplayGroupEntity> displayGroups) {
         }
         displayGroupRepository.saveAll(displayGroups);
     }
+
+    private void validate(List<GenericLayoutEntity> genericLayouts) {
+        ValidationResult result = new ValidationResult();
+
+        for (GenericLayoutValidator validator : genericLayoutValidators) {
```

Comment:
any reason why we changed the way we did validation? i.e. before we were looping through the fields first and applying for each field the validators. And this is what we do for all the validations also on other types of fields. But here we swapped it. We loop through the validators, and then each validator loops through all the fields. I think the other way made more sense